pkg/sfu/forwarder.go

```diff
@@ -385,11 +385,13 @@ func (f *Forwarder) GetState() *livekit.RTPForwarderState {
 	state := &livekit.RTPForwarderState{
 		Started:                   f.started,
 		ReferenceLayerSpatial:     f.referenceLayerSpatial,
-		PreStartTime:              f.preStartTime.UnixNano(),
 		ExtFirstTimestamp:         f.extFirstTS,
 		DummyStartTimestampOffset: f.dummyStartTSOffset,
 		RtpMunger:                 f.rtpMunger.GetState(),
 	}
+	if !f.preStartTime.IsZero() {
+		state.PreStartTime = f.preStartTime.UnixNano()
+	}
```

Comment:
Converting zero time to UnixNano() and converting back to `time.Time` ended up with `IsZero() = false`. And that caused time stamp offset to get messed up after seeding. So, check for 0 before conversion.